added cover ui, installation-cover relation

tudorbarascu opened the pull request:
Looking like this:

![cover](https://cloud.githubusercontent.com/assets/3179646/13532239/0bb60b90-e233-11e5-91be-fa44181bce5b.png)

3nids commented on qgis-project/qwat.qgs:
```diff
@@ -88941,18 +88944,7 @@ def my_form_open(dialog, layer, feature):
-WHEN  "installation_type"  = 'treatment' THEN 'T. '
-ELSE ''
-END
-||name</previewExpression>
```
I know it's in french, but I need it in our project at this stage...

tudorbarascu replied:
I haven't removed the expression from the label, but from the relation widget on fk_installation so that it shows the installation id and not what what's in ('CR.','CCP','CRA' etc.).

So, what's presented here:
![cover](https://cloud.githubusercontent.com/assets/3179646/13568981/e360f2b2-e46d-11e5-8f74-a73cea86d9c9.png)

Are you ok with it or do you want me to have in the widget the same expression in the label?

3nids commented:
The expression was giving type + name, I would still need it.
sorry for the incovenience

tudorbarascu commented:
Cool, the expression I had changed was for the fk_installation widget, so that it outputted the id of the installation. I made it now like you desired and I also fixed a problem where the labels didn't show when the name of the installation was NULL.

The expression ends now with ``||coalesce(name,'')``

The output of the fk_installation widget on the cover layer is:
![cover1](https://cloud.githubusercontent.com/assets/3179646/13573553/e5c153b0-e488-11e5-96a0-519c0d3c376d.png)

Is this fine for you or an id of the installation would have also been good there?
All the best Denis, no inconvenience.

tudorbarascu commented:
@3nids It should be good now, could you please take a look?

There was an issue with labels that didn't show when the name of the installation was NULL I fixed it in the second commit. If you think that's unnecessary I can just remove that commit and rebase.

Thanks!

3nids commented:
perfect, thanks
